### DISCUSSION TITLE
How to register neoforge to overwrite the vanilla registration? Is there such a possibility?

### DISCUSSION THREAD

**Discussion by baka4n** [neoforged/NeoForge#161] [Support]
Now I want to register the block for the vinalla food and the block has its own modid, and then the mixin redirects the item, which will return my block is null

**Comment by TelepathicGrunt** [comment]
Have you tried the registry replacement that vanilla registries allow? Though generally, most things people want to do for vanilla’s stuff doesn’t need registry replacement so if you can state your endgoal, we might be able to tell you an alternative

**Comment by baka4n** [comment]
I'm working on a piece of rustic food and rustic decoration, but I really need to write about the placement of the food piece, because there is a food pile on the ground, but with the vanilla square, it is definitely better to register multiple events

**Comment by baka4n** [comment]
Use BlockItem instead of Item
